rdr_service/alembic/versions/56725021b7b3_.py

```diff
@@ -0,0 +1,61 @@
+"""
+
+Revision ID: 56725021b7b3
+Revises: f46e62e1a637
+Create Date: 2021-04-16 15:55:12.079922
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import rdr_service.model.utils
+
+
+from rdr_service.participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from rdr_service.participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from rdr_service.participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from rdr_service.participant_enums import OrderShipmentTrackingStatus, OrderShipmentStatus
+from rdr_service.participant_enums import MetricSetType, MetricsKey, GenderIdentity
+from rdr_service.model.base import add_table_history_table, drop_table_history_table
+from rdr_service.model.code import CodeType
+from rdr_service.model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+
+# revision identifiers, used by Alembic.
+revision = '56725021b7b3'
+down_revision = 'f46e62e1a637'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute(
+        "Update genomic_incident Set code = 'FILE_VALIDATION_FAILED_FIELDS' WHERE code = 'FILE_VALIDATION_FAILED'"
```

Comment:
Any reason why this needs to be in a migration?

Reply:
Just figured it was easier than overwriting it now and then overwriting it again when the enums changes get to prod, since the timelapse will be around 2 weeks before this PR makes it to prod.

Reply:
Can remove if needed

Reply:
No, that makes sense.

Reply:
Is FILE_VALIDATION_FAILED_FIELDS intentionally distinct from any of the options in the `GenomicIncidentCode` enum, or should it be one of "FILE_VALIDATION_FAILED_NAME" or "FILE_VALIDATION_FAILED_STRUCTURE" instead?